pkg/sidecar/config.go

```diff
@@ -5,7 +5,6 @@ import (
 )
 
 type Config struct {
-	// If true, merge intermediate certificates into Bundle file instead of SVID file.
```

Comment:
How is this change related? Was it's removal intentional?

Reply:
accident

Reply:
added back in